infoschema/tables.go

```diff
@@ -1850,6 +1850,24 @@ func GetPDServerInfo(ctx sessionctx.Context) ([]ServerInfo, error) {
 	return servers, nil
 }
 
+func isTiFlashStore(store *metapb.Store) bool {
+	for _, label := range store.Labels {
+		if label.GetKey() == placement.EngineLabelKey && label.GetValue() == placement.EngineLabelTiFlash {
```

Comment:
Should we also check EngineLabelTiFlashCompute?

Reply:
tiflash_compute is a store type in client-go, so no need to check here.